detox/src/configuration/composeBehaviorConfig.js

```diff
@@ -16,8 +16,7 @@ function composeBehaviorConfig({
   isCloudSession
 }) {
   if (isCloudSession) {
-    cliConfig.reuse = false;
-    cliConfig.cleanup = false;
+    cliConfig.reuse = true;
```

Comment:
Changed default value for reuse to `true` and shutdownDevice to `false` for cloud sessions